app/src/main/java/com/gruppo3/wetravel/mapManager/types/ViewMap.java

```diff
@@ -44,17 +44,19 @@ public ViewMap() {
 
     /**
      * Creates a new instance of {@link ViewMap} with given {@link LocationRequest} and map zoom level parameters.
-     * @param locationRequest Object of type LocationRequest with custom request options.
-     * @param mapZoom Map zoom level.
+     *
+     * @param locationRequest Object of type {@link LocationRequest} with custom request options.
+     * @param mapZoom         Map zoom level.
      */
     public ViewMap(@NonNull LocationRequest locationRequest, int mapZoom) {
         this.locationRequest = locationRequest;
         this.mapZoom = mapZoom;
     }
 
     /**
-     * Creates a new instance of ViewMap with given LocationRequest parameter and default zoom level.
-     * @param locationRequest Object of type LocationRequest with custom parameters.
+     * Creates a new instance of ViewMap with given {@link LocationRequest} parameter and default zoom level.
```

Comment:
You might have missed this, since you used it in all constructors.
```suggestion
     * Creates a new instance of {@link ViewMap} with given {@link LocationRequest} parameter and default zoom level.
```